tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+"""Pytest fixtures."""
+
+from enum import Enum
+from typing import cast
+
+import pytest
+
+from array_api_extra._lib._compat import array_namespace
+from array_api_extra._lib._compat import device as get_device
+from array_api_extra._lib._typing import Device, ModuleType
+
+
+class Library(Enum):
+    """All array libraries explicitly tested by array-api-extra."""
+
+    ARRAY_API_STRICT = "array_api_strict"
+    NUMPY = "numpy"
+    NUMPY_READONLY = "numpy_readonly"
+    CUPY = "cupy"
+    TORCH = "torch"
+    DASK_ARRAY = "dask.array"
+    SPARSE = "sparse"
```

Comment:
For the record: there are ZERO tests in the whole test suite that run successfully on sparse, for a wide variety of reason.